web_widget_timepicker/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2016 Vividlab (<http://www.vividlab.de>)
```

Comment:
(c) -> Copyright

Reply:
I'm not Vividlab... idk if I can change this... :S

Reply:
You can just replace © with 'Copyright'. I don't think it's a problem :smile: